client/components/jetpack/portal-nav/index.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { useSelector } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { isSectionNameEnabled } from 'calypso/sections-filter';
+import {
+	getAllPartners,
+	hasFetchedPartners,
+	isPartnerPortal,
+} from 'calypso/state/partner-portal/selectors';
+import QueryJetpackPartnerPortalPartners from 'calypso/components/data/query-jetpack-partner-portal-partners';
+import SectionNav from 'calypso/components/section-nav';
+import NavTabs from 'calypso/components/section-nav/tabs';
+import NavItem from 'calypso/components/section-nav/item';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Props {
+	className: string;
+}
+
+function PortalNav( { className }: Props ) {
+	const translate = useTranslate();
+	const partnersFetched = useSelector( hasFetchedPartners );
+	const partners = useSelector( getAllPartners );
+	const isManagingSites = ! useSelector( isPartnerPortal );
+	const selectedText = isManagingSites
+		? translate( 'Manage Sites' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Manage sites I follow' )` *ES Score: 6*